pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -61,13 +61,13 @@ func (r *resourceAllocationScorer) score(
 	var score int64
 
 	// Check if the pod has volumes and this could be added to scorer function for balanced resource allocation.
-	if len(pod.Spec.Volumes) >= 0 && utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) && nodeInfo.TransientInfo != nil {
```

Comment:
I guess it's meant to be `len(pod.Spec.Volumes) > 0`?

Reply:
done